2-mandatory/3-financial-times.js

```diff
@@ -23,59 +32,84 @@ function titleWithFewestWords(allArticleTitles) {
     (Hint: remember that you can also loop through the characters of a string if you need to)
 */
 function headlinesWithNumbers(allArticleTitles) {
-    // TODO
+  let arr = [];
+
+  for (let moreClick of allArticleTitles) {
+    let choosen = "";
+    for (let letter of moreClick) {
+      if (!isNaN(letter) && letter !== " ") {
+        choosen = moreClick;
+      }
+    }
+    if (choosen.length !== 0) {
+      arr.push(choosen);
+    }
+  }
+  return arr;
```

Comment:
Nice work! You may consider adding a break after line 41 to avoid "shortcut" the function. Once you find a number in the article. You can put it in the final array and jump to process another article.

Reply:
done :)